File: nelio-alves/algoritmo-e-logica-de-programacao/c/exercicios/media_pares.c
#include <stdio.h>

int main() {
    int N, soma, cont;
    double media;

    printf("Quantos elementos vai ter o vetor? ");
    scanf("%d", &N);

    int vet[N];

    for (int i = 0; i < N; i++) {
        printf("Digite um numero: ");
        scanf("%d", &vet[i]);
    }

    cont = 0;
    soma = 0;
    for (int i = 0; i < N; i++) {
        if (vet[i] % 2 == 0) {
            soma = soma + vet[i];
            cont++;
        }
    }

    if (cont == 0) {
        printf("NENHUM NUMERO PAR\n");
    } else {
        media = (double) soma / cont;

        printf("MEDIA DOS PARES = %.1lf\n", media);
    }

    return 0;
}

File: nelio-alves/algoritmo-e-logica-de-programacao/java/src/exercicios/MediaPares.java
package exercicios;

import java.util.Locale;
import java.util.Scanner;

public class MediaPares {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int N, soma, cont;
		double media;

	    System.out.print("Quantos elementos vai ter o vetor? ");
		N = sc.nextInt();

	    int[] vet = new int[N];

		for (int i = 0; i < N; i++) {
	    	System.out.print("Digite um numero: ");
			vet[i] = sc.nextInt();
	    }

		cont = 0;
		soma = 0;
		for (int i = 0; i < N; i++) {
			if (vet[i] % 2 == 0) {
				soma = soma + vet[i];
				cont++;
			}
		}

		if (cont == 0) {
			System.out.println("NENHUM NUMERO PAR");
		} else {
			media = (double) soma / cont;

			System.out.printf("MEDIA DOS PARES = %.1f\n", media);
		}

		sc.close();
	}
}
